apiserver/facades/client/bundle/bundle_test.go

```diff
@@ -709,3 +762,49 @@ func (s *bundleSuite) TestExportBundleModelWithAnnotations(c *gc.C) {
 	c.Assert(result, gc.Equals, expectedResult)
 	s.st.CheckCall(c, 0, "ExportPartial", s.st.GetExportConfig())
 }
+
+func (s *bundleSuite) TestExportBundleWithContainers(c *gc.C) {
```

Comment:
Could we extend this test to also include an application on a non-container machine. So we'll end up with machine 0 and 1 and container lxd:0

Reply:
Done.